nodebuilder/config.go

```diff
@@ -80,6 +81,48 @@ func LoadConfig(path string) (*Config, error) {
 	return &cfg, cfg.Decode(f)
 }
 
+// RemoveConfig removes the Config from the given store path.
+func RemoveConfig(storePath string) error {
+	return removeConfig(configPath(storePath))
+}
+
+// removeConfig removes Config from the given 'path'.
+func removeConfig(path string) error {
+	return os.Remove(path)
+}
+
+// UpdateConfig loads the node's config and applies new values
+// from the default config of the given node type, saving the
+// newly updated config into the node's config path.
+func UpdateConfig(tp node.Type, storePath string) error {
```

Comment:
For both Remove and Update we should lock the repo, so that we are not doing dirty reads/writes

Reply:
done